examples/compressible/skamarock_klemp_nonhydrostatic.py

```diff
@@ -69,13 +69,11 @@
     output = OutputParameters(
         dirname=dirname,
         dumpfreq=dumpfreq,
-        pddumpfreq=dumpfreq,
-        dumplist=['u'],
-    )
+        dump_nc=True,
+        dump_vtus=False
+          )
 
-diagnostic_fields = [CourantNumber(), Gradient("u"), Perturbation('theta'),
-                     Gradient("theta_perturbation"), Perturbation('rho'),
-                     RichardsonNumber("theta", parameters.g/Tsurf), Gradient("theta")]
+diagnostic_fields = [CourantNumber(), Pressure(eqns), SteadyStateError('Pressure_Vt')]
```

Comment:
Are any of the changes to this file meant to be added to this PR?

Reply:
yes im not sure why the examples have been added as i was unaware that these had even been pushed from my local area?

Reply:
Removed the changes to the skamarock test and implemented it in Williamson2